doc/source/user_guide/categorical.rst

```diff
@@ -439,15 +439,19 @@ Sorting and order
 .. _categorical.sort:
 
 If categorical data is ordered (``s.cat.ordered == True``), then the order of the categories has a
-meaning and certain operations are possible. If the categorical is unordered, ``.min()/.max()`` will raise a ``TypeError``.
+meaning and certain operations are possible. If the categorical is unordered, the data can still be sorted,
+but ``.min()/.max()`` will raise a ``TypeError``.
 
 .. ipython:: python
 
     s = pd.Series(pd.Categorical(["a", "b", "c", "a"], ordered=False))
-    s.sort_values(inplace=True)
-    s = pd.Series(["a", "b", "c", "a"]).astype(CategoricalDtype(ordered=True))
-    s.sort_values(inplace=True)
-    s
+    s.sort_values(ascending=False)
+    try:
+        s.min()
+    except TypeError as e:
+        print(f"TypeError: {e}")
+    s = s.astype(CategoricalDtype(ordered=True))
+    s.sort_values()
```

Comment:
I think this will need to be `s = s.sort_values()`

Reply:
Of course you could write `s = s.sort_values()` followed by `s` to show the result but as we don't need the sorted series further down (I mean for the following code it doesn't play a role if `s` has been sorted or not) I think this could be shortened to just `s.sort_values()` which will output the returned sorted Series.  

In principle, one could remove this line completely, because if we can sort an unordered categories series, we can sort an ordered categories series all the more (it's a leftover from the beginning, when sorting of an unordered categories series was not allowed, this was changed in 87fec5b527dcbb).

Reply:
Sure removing this line would be better then